portable/Common/mpu_wrappers_v2.c

```diff
@@ -112,6 +112,16 @@
  */
     #define CONVERT_TO_INTERNAL_INDEX( lIndex )    ( ( lIndex ) - INDEX_OFFSET )
 
+/**
+ * @brief Max value that fits in a size_t type.
+ */
+    #define mpuSIZE_MAX    ( ~( ( size_t ) 0 ) )
```

Comment:
I would consider changing this to uint32_t to match the type of parameter 2 in MPU_uxTaskGetSystemStateImpl()